apis/r/DESCRIPTION

```diff
@@ -6,7 +6,7 @@ Description: Interface for working with 'TileDB'-based Stack of Matrices,
     like those commonly used for single cell data analysis. It is documented at
     <https://github.com/single-cell-data>; a formal specification available is at
     <https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md>.
-Version: 1.15.99.20
+Version: 1.16.99.20
```

Comment:
I thought I had 1.16.99

Let me check to see if I missed pushing a sandbox mod to GitHub ...

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/3462/commits/ff638071b71396d1c619a730f8c8c91cc67947eb

This is what we agreed on collectively and as a team here:
https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases

If you don't want that -- if you want 1.16.99.0 rather than 1.16.99 -- let's re-open that team discussion and re-acquire consensus and update the procedure.

Reply:
1.16.99 is good and equivalent to 1.16.99.0, the main thing was resetting the development version, so thanks for taking care of that

Reply:
Thanks @mojaveazure !